### PR TITLE
Update manifest.yaml

### PR DESCRIPTION
**Pull request by kn0wmad**

### CODE REVIEW

File: manifest.yaml [hunk 1]
```diff
@@ -141,7 +141,7 @@ backup:
 actions:
   reindex:
     name: "Reindex Blockchain"
-    description: "Rebuilds the block and chainstate databases starting from genesis. If blocks already exist on disk, these are used rather than being redownloaded. However, since embassy bitcoin nodes are pruned by default, this usually means downloading the entire blockchain over again."
+    description: "Rebuilds the block and chainstate databases starting from genesis. If blocks already exist on disk, these are used rather than being redownloaded. This usually means downloading the entire blockchain over again, so please be prepared for this inconvenience."
```
Comment (chrisguida) [inline manifest.yaml:144]: This actually isn't true for archival nodes

**Review by chrisguida — CHANGES_REQUESTED** [review]
see comment

File: manifest.yaml (hunk 1 above)
Comment (kn0wmad) [reply manifest.yaml:144]: What is it that is happening when an archival is reindexed?  It seems to take as long as a full IBD

Comment (chrisguida) [reply manifest.yaml:144]: It's re-validating all blocks from disk, and reinserting them into the index.

Comment (chrisguida) [reply manifest.yaml:144]: Basically, everything IBD does _except_ redownload the blocks

**Comment by kn0wmad** [comment]
Okay, so what is the final copy change here then?  I feel like I should just assign to you as you probably could have written it in 20 seconds @chrisguida

**Comment by chrisguida** [comment]
@kn0wmad see my code suggestion for what I think it should be. This is your issue, so you can decide whether it closes your issue

**Comment by chrisguida** [comment]
Closes #90

**Pull request merged**